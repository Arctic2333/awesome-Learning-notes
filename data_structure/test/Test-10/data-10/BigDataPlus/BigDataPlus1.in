123456789012345678901234567890
999999999999999999999999999999999
