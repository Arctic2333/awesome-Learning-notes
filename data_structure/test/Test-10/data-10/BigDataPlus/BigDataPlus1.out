1000123456789012345678901234567889